entity-service-api/src/main/proto/org/hypertrace/entity/query/service/v1/entity_query_request.proto

```diff
@@ -12,7 +12,7 @@ message EntityQueryRequest {
   string entityType = 2;
   Filter filter = 3;
   repeated Expression selection = 4;
-
+  repeated OrderByExpression orderBy = 7;
```

Comment:
You're saying you don't want to reuse b/w EDS and EQS but you're reusing it here right?

Reply:
It is not, it's using the OrderByExpression definition from query/service/v1/request.proto. 
Definition is : 
`message OrderByExpression {
  Expression expression = 1;
  SortOrder order = 2;
}
`
While in EDS it uses the one in data/service/v1/entity_data_request.proto. The definition is : 
```
message OrderByExpression {
  string name = 1;
  SortOrder order = 2;
}
```